plugins/flow-visibility/clickhouse-monitor/main.go

```diff
@@ -58,113 +50,37 @@ var (
 	tableName = os.Getenv("TABLE_NAME")
 	// The names of the materialized views
 	mvNames = strings.Split(os.Getenv("MV_NAMES"), " ")
-	// The namespace of the ClickHouse server
-	namespace = os.Getenv("NAMESPACE")
-	// The ClickHouse monitor label
-	monitorLabel = os.Getenv("MONITOR_LABEL")
+	// The remaining number of rounds to be skipped
+	remainingRoundsNum = 0
+	// The monitor runs every minute.
+	monitorTicker = time.NewTicker(1 * time.Minute)
 )
 
 func main() {
 	// Check environment variables
-	if len(tableName) == 0 || len(mvNames) == 0 || len(namespace) == 0 || len(monitorLabel) == 0 {
-		klog.ErrorS(nil, "Unable to load environment variables, TABLE_NAME, MV_NAMES, NAMESPACE and MONITOR_LABEL must be defined")
+	if len(tableName) == 0 || len(mvNames) == 0 {
+		klog.ErrorS(nil, "Unable to load environment variables, TABLE_NAME and MV_NAMES must be defined")
 		return
 	}
-	// The monitor stops working for several rounds after a deletion
-	// as the release of memory space by the ClickHouse MergeTree engine requires time
-	if !skipRound() {
-		connect, err := connectLoop()
-		if err != nil {
-			klog.ErrorS(err, "Error when connecting to ClickHouse")
-			return
-		}
-		deleted := monitorMemory(connect)
-		if deleted {
-			klog.InfoS("Skip rounds after a successful deletion", "skipRoundsNum", skipRoundsNum)
-		} else {
-			klog.InfoS("Next round will not be skipped", "skipRoundsNum", 0)
-		}
-	}
-}
 
-// Checks the k8s log for the number of rounds to skip.
-// Returns true when the monitor needs to skip more rounds and log the rest number of rounds to skip.
-func skipRound() bool {
-	logString, err := getPodLogs()
+	connect, err := connectLoop()
 	if err != nil {
-		klog.ErrorS(err, "Not find last monitor job")
-		return false
+		klog.Fatalf("Error when connecting to ClickHouse: %v", err)
```

Comment:
since we are trying to only use the structured logging functions, maybe replace this with `klog.ErrorS` + `os.Exit(1)`

Reply:
Replaced